elasticdl/proto/elasticdl.proto

```diff
@@ -41,6 +47,9 @@ message Task {
 
     // Whether this is training or evaluation task.
     TaskType type = 7;
+
+    // Extended task configuration as a key value list
+    repeated Pair extended_config = 8;
```

Comment:
Add comment here about the info that are stored in `extended_config`?

Reply:
Good suggestion, added.